test/test_helpers.py

```diff
@@ -432,6 +432,34 @@ def test_redq_make(device, from_pixels, gsde, exploration):
     del proof_environment
 
 
+def test_seed_generator():
```

Comment:
great test function

Perhaps let's move the loop out of it, and do something like

```
@pytest.mark.parameterize("initial_seed", range(5))
def test_seed_generator(initial_seed):
    ...
```